src/components/pages/CustomerDashboard/cust-tabs.js

```diff
@@ -25,12 +27,18 @@ const CustTab = () => {
   const { resultInfo } = useContext(FormContext);
   const { custInfo } = useContext(CustomersContext);
   const { getCustomerByID } = useContext(APIContext);
+  const { getPet } = useContext(APIContext);
+  // const { pets } = useContext(PetsContext); // testing that context is updated before pets are rendered in RenderPetProfile
 
   useEffect(() => {
     getCustomerByID(authState);
     // eslint-disable-next-line react-hooks/exhaustive-deps
+    getPet(authState);
   }, []);
 
+  // testing that context is updated before pets are rendered in RenderPetProfile
+  // console.log('cust-tabs: ', pets)
```

Comment:
make sure to take out any console log that is in the code. It is easily put back if we need to test something.

Reply:
This is done. The commented-out lines were removed.